src/Commands/ModuleDownloadCommand.php

```diff
@@ -0,0 +1,359 @@
+<?php namespace Wireshell\Commands;
+
+use Distill\Distill;
+use Distill\Exception\IO\Input\FileCorruptedException;
+use Distill\Exception\IO\Input\FileEmptyException;
+use Distill\Exception\IO\Output\TargetDirectoryNotWritableException;
+use Distill\Strategy\MinimumSize;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Message\Response;
+use GuzzleHttp\Subscriber\Progress\Progress;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Wireshell\PwConnector;
+
+/**
+ * Class ModuleDownloadCommand
+ *
+ * Downloads module(s)
+ *
+ * @package Wireshell
+ * @author Marcus Herrmann
+ * @author Tabea David <td@kf-interactive.com>
+ */
+
+class ModuleDownloadCommand extends PwConnector
+{
+
+
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    protected static $defaults = array(
+        'apikey' => 'pw223',
```

Comment:
Ryan suggested to use $config->moduleServiceURL and $config->moduleServiceKey instead. Since c51d62d22b4b1a45e998dc8501b0bd56125c90cb (on develop branch) these are available for wireshell commands that extend PwConnector with $this->moduleServiceKey and $this->moduleServiceKey :)